entity-attributes-card/entity-attributes-card.js

```diff
@@ -5,34 +5,58 @@ class EntityAttributesCard extends HTMLElement {
   }
 
   _getAttributes(hass, filters) {
-    function _filterName(stateObj, pattern) {
+    function _getPart(pattern) {
       let parts;
-      let attr_id;
       let attribute;
       if (typeof (pattern) === "object") {
         parts = pattern["key"].split(".");
         attribute = pattern["key"];
-        
       } else {
         parts = pattern.split(".");
         attribute = pattern;
       }
-      attr_id = parts[2];
-      if (attr_id.indexOf('*') === -1) {
-        return stateObj == attribute;
+      return { parts: parts, attr: attribute, 
+	       match_by_entity: parts[1].indexOf('*') != -1, 
+	       match_by_attr: parts[2].indexOf('*') != -1
+      };
+    }
+
+    function _filterName(stateObj, pattern) {
+      const res = _getPart(pattern);
+      let attr_id = res.parts[2];
+      let entity_id = res.parts[1];
+      if (attr_id.indexOf('*') === -1 && entity_id.indexOf('*') === -1) {
+        return stateObj == res.attr;
       }
-      const regEx = new RegExp(`^${attribute.replace(/\*/g, '.*')}$`, 'i');
+      const regEx = new RegExp(`^${res.attr.replace(/\*/g, '.*')}$`, 'i');
       return stateObj.search(regEx) === 0;
     }
+
     const attributes = new Map();
     filters.forEach((filter) => {
       const filters = [];
       filters.push(stateObj => _filterName(stateObj, filter));
```

Comment:
the _getPart should be added here not inside an existing filter. Make sure they execute in correct order. I might not understand completely what you're trying to achieve. If you could please include some configuration as well with the picture.